### ISSUE TITLE
Using '32-bit libraries for X' as the summary for a package doesnt always work

### ISSUE THREAD

**Issue by philipzae** [solus-project/solus-sc#122]
This doesnt work well when the package is a 32-bit build of an app and not a library.
Some examples include util-linux-32bit, xz-32bit, wine-32bit,

**Comment by ikeydoherty** [comment]
32-bit libraries by themselves are never build dependencies. `-32bit-devel` packages _are_. For this mechanism we provide the `pkgconfig32()` ypkg builddep.

The only example there I'd be willing to alter is `wine-32bit` which, unlike basically every other package, contains a `wine` (32-bit) and `wine64` (64-bit) set of binaries. Outside of that I'm not really sure I see your point.

**Comment by philipzae** [comment]
Yes my point was to correct summaries from '32-bit libraries for X' to '32-bit binaries for X' for 32-bit applications. If by searching for '32bit' in the software center only wine-32bit deserves to be changed, thats fine.

**Comment by ikeydoherty** [comment]
Yeah typically nobody really needs to search for -32bit stuff, they're deps of real things. Searching for wine will yield wine-32bit.
